tests/ecosystem/upgrade/test_upgrade.py

```diff
@@ -232,10 +232,8 @@ def test_upgrade():
         )
         if subscription_plan_approval == 'Manual':
             wait_for_install_plan_and_approve(namespace)
-        attempts = 145
-        for attempt in range(1, attempts):
-            if attempts == attempt:
-                raise TimeoutException("No new CSV found after upgrade!")
+        attempts = 10
```

Comment:
Why are we going from 145 attempts to 10?

Reply:
good catch, was testing with lower value :), will fix.